src/applications/disability-benefits/all-claims/tests/pages/contactInformation.unit.spec.jsx

```diff
@@ -10,13 +10,13 @@ import { combineReducers, createStore } from 'redux';
 
 import {
   DefinitionTester, // selectCheckbox
```

Comment:
Wondering if the `selectCheckbox` import should just be removed if it's disabled and not being called in the spec.